src/pages/teacherProfile.tsx

```diff
@@ -522,35 +389,38 @@ const MyProfile = () => {
             >
               <Box
                 sx={{
-                  //flex: '1',
+                  flex: '1',
                   textAlign: 'center',
                   marginLeft: '5%',
+                  alignItems: 'center',
+                  justifyContent: 'center',
                 }}
                 borderRadius={'12px'}
                 border={'1px'}
                 bgcolor="warning.A400"
                 display="flex"
                 flexDirection="column"
               >
-                <Image
-                  src={prathamProfileUrl}
+                <img
+                  src={image}
```

Comment:
@upendraTekdi use Image component from the Next

Reply:
okay